src/cascadia/TerminalApp/App.xaml

```diff
@@ -44,6 +44,11 @@ the MIT License. See LICENSE in the project root for license information. -->
                         <Setter Target="CloseButtonStyle" Value="{StaticResource AccentButtonStyle}" />
                     </Style>
 
+                    <!-- We need to manually create the error text brush as a
+                    theme-dependent brush. SystemControlErrorTextForegroundBrush
+                    is unfortunately static. -->
+                    <SolidColorBrush x:Name="ErrorTextBrush" Color="{ThemeResource SystemErrorTextColor}" />

```

Comment:
Hmm, does this work if you runtime-switch from light to dark?

Reply:
If not we may need to put it _in_ the theme dictionaries.

Reply:
Well, actually the entire dialog doesn't runtime switch ATM so it's kinda for the best that this doesn't :P